docs/shared_parsers_catalog/dracut_modueles.rst

```diff
@@ -0,0 +1,3 @@
+.. automodule:: insights.parsers.dracut_modueles
```

Comment:
I believe the name of the module should be `dracut_modules`.

BTW, would you please consider rename it accordingly?